packaging/linux/build_binaries.sh

```diff
@@ -36,13 +36,19 @@ fi
 echo "-tags '$go_tags'"
 
 # Determine the LD flags.
-ldflags=""
+ldflags_client=""
+ldflags_kbfs=""
 if [ "$mode" != "production" ] ; then
   # The non-production build number is everything in the version after the hyphen.
   build_number="$(echo -n "$version" | sed 's/.*-//')"
-  ldflags="-X github.com/keybase/client/go/libkb.PrereleaseBuild=$build_number"
+  ldflags_client="-X github.com/keybase/client/go/libkb.PrereleaseBuild=$build_number"
+  kbfs_path="$here/../../../kbfs"
+  commit_short_kbfs="$(git -C "$kbfs_path" log -1 --pretty=format:%h)"
+  build_number_kbfs="$(echo -n "$build_number" | sed 's/+..*/+/')"$commit_short_kbfs
```

Comment:
Let's go ahead and put `$commit_short_kbfs` inside the closing `"`, just to be paranoid about bash whitespace.